source/fundamentals/crud/read-write-pref.txt

```diff
@@ -0,0 +1,237 @@
+levels:
+
+- Client, which sets the *default for all operation executions* unless overridden
+- Session
+- Transaction
+- Database
+- Collection
+
+These options allow you to customize the consistency and availability of the data in your
+replica sets.
+
+Write Concern
+-------------
+
+The write concern specifies the level of acknowledgement requested from MongoDB for write
+operations, such as an insert or update, before the operation successfully returns.
+Operations that do not specify an explicit write concern inherit the global default write
+concern settings.
+
+For more information, see :manual:`Write Concern </reference/write-concern/>` in the
+Server manual. For detailed API documentation, see the `WriteConcern API documentation
+<{+api+}/classes/WriteConcern.html>`__.
+
+The following table describes the ``WriteConcern`` parameters:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 25 50
+
+   * - Parameter
+     - Type
+     - Description
+  
+   * - ``w`` *(optional)*
+     - `W <{+api+}/types/W.html>`__
+     - Requests acknowledgment that the write operation has propagated to a specified number of mongod instances or to mongod instances with specified tags
+
+   * - ``wtimeoutMS`` *(optional)*
+     - number
+     - Specifies a time limit to prevent write operations from blocking indefinitely
+
+   * - ``journal`` *(optional)*
+     - boolean
+     - Requests acknowledgment that the write operation has been written to the on-disk journal 
+
+Example: Set the Write Concern for a Single Write Operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This code uses custom ``WriteConcern`` settings while creating new a document:
+
+.. code-block:: js
+
+   myDB.myCollection.insertOne( 
+      { name: "anotherDocumentName" }, 
+      { writeConcern: 
+        { w: 2, wtimeoutMS: 5000 } 
+      } 
+   );
+
+Example: Retrieve and Apply an Existing Write Concern
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+   
+This code uses the ``fromOptions()`` method to construct a ``WriteConcern`` from the
+options of an existing database, ``myDB``. Note that ``myDB`` could be replaced with any
+entity that accepts a write concern option. Then the new write concern is applied to a
+document, ``myDoc``.
+
+.. code-block:: js
+
+   const newWriteConcern = WriteConcern.fromOptions(myDB);
+   const myDoc = { name: "New Document" };
+   WriteConcern.apply(myDoc,newWriteConcern);
+
+Read Concern
+------------
+
+The read concern specifies the following behaviors:
+
+- Level of :manual:`causal consistency </core/causal-consistency-read-write-concerns/>` across replica sets
+
+- `Isolation guarantees
+  <https://www.mongodb.com/docs/current/core/read-isolation-consistency-recency/>`__ maintained during a query 
+
+Lower consistency requirements may reduce latency. The default read concern level is
+``local``. This means that the client returns the data from the replica set member that the
```

Comment:
Suggestion:

I think it could be helpful to identify that the setting parameter is called "level" prior to the example. The Read Preference and Write Concern sections both include the parameter name (in a table).

"You can specify the read concern setting in the \`\`level\`\` parameter. The default..."